service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadCMOSubmittedControllerTest.java

```diff
@@ -100,7 +102,21 @@ void shouldSendNotificationsIfNewCMOUploaded() {
         });
     }
 
-    private CallbackRequest callbackRequest() {
+    @Test
+    void shouldSendToJudgeIfDraftCMOUploaded() {
+        CallbackRequest callbackRequest = callbackRequest(DRAFT);
+
+        postSubmittedEvent(callbackRequest);
+
+        checkUntil(() -> verify(notificationClient).sendEmail(
+            eq(CMO_DRAFT_UPLOADED_NOTIFICATION_TEMPLATE),
+            eq(JUDGE_EMAIL),
+            anyMap(),
```

Comment:
are we able to get the information what mapping we are sending to the template?

Reply:
We would know the information being sent but do we need to construct it or can we trust our unit tests that the right data will have been passed?

Reply:
if you knew that all the parameters were correct you could put anyString(),anyString(),anyMap(), but since we have the data, an extra assertion would just give more confidence in the assertions. That's one of the only things that we can assert on integration/black box tests, if you think as the client as our only output/verifiable interaction.

Reply:
Updated to assert against template, means I no longer need the `eq(...)` as well which is nice